.changelog/config.toml

```diff
@@ -0,0 +1,20 @@
+project_url = "https://github.com/cosmos/gaia"
```

Comment:
This is here because we're not backporting the addition of `unclog` to main onto this branch?

Reply:
Yes.